scipy/spatial/distance.py

```diff
@@ -1856,8 +1895,8 @@ class MetricInfo:
         canonical_name='jensenshannon',
         aka={'jensenshannon', 'js'},
         dist_func=jensenshannon,
-        cdist_func=CDistMetricWrapper('jensenshannon'),
-        pdist_func=PDistMetricWrapper('jensenshannon'),
+        cdist_func=_distance_pybind.cdist_jensenshannon,
```

Comment:
When I run the benchmarks I'm seeing ~50% slow down in unweighted jensenshannon from this PR.